win32ss/gdi/ntgdi/text.c

```diff
@@ -525,7 +527,7 @@ NtGdiGetTextFaceW(
             return 0;
         }
         /* Terminate if we copied only part of the font name */
-        if (Count > 0 && Count < fLen)
+        if (0 < Count && Count <= fLen)
         {
             FaceName[Count - 1] = '\0';
```

Comment:
Naive question: We don't need SEH here?

Reply:
MmCopyToCaller does it.

Reply:
Yeah except you need SEH every time you access user memory, because another thread could be deallocating it concurrently. So yes, we need SEH here.

Reply:
How to do?

Reply:
OK, SEH handled.